src/settings.rs

```diff
@@ -0,0 +1,73 @@
+//use crate::LOG_DRAIN;
+
+use serde::{Deserialize, Serialize};
+//use slog::info;
+
+// Describe the settings your policy expects when
+// loaded by the policy server.
+#[derive(Serialize, Deserialize, Default, Debug)]
+#[serde(default)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct Settings {
+    pub reserved_runtimes: Vec<String>,
```

Comment:
I would use a `Set` instead, this provides better performance when doing lookups (like we do inside of validate)